.github/workflows/dependabot-auto-merge.yaml

```diff
@@ -0,0 +1,28 @@
+---
+
+name: Auto merge Dependabot updates
+
+on:
+  check_suite:
+    types:
+      - completed
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+
+jobs:
+  auto-merge:
+    name: Auto merge
+    runs-on: ubuntu-latest
```

Comment:
probably not important but this one uses `ubuntu-latest` and the other `ubuntu-18.04`